contract_variable_quantity/models/contract.py

```diff
@@ -13,3 +13,13 @@ class ContractContract(models.Model):
         string="Skip Zero Qty Lines",
         help="If checked, contract lines with 0 qty don't create invoice line",
     )
+
+    def _prepare_recurring_invoices_values(self, date_ref=False):
+        vals = super()._prepare_recurring_invoices_values(date_ref=date_ref)
```

Comment:
I don't get why are you doing this, but it seems very incorrect, as this module only operates in the quantity field of the invoice line, not on invoice header.

Reply:
I do it because when `_prepare_invoice_line` return `vals` as `{}`. This method create a line on the contract, I do it for avoid this line creation. Based on test, I think that's the original scope...